src/docs/variant_relative_location.md

```diff
@@ -5,18 +5,26 @@ It is important to know where in a genomic feature a variant lies and in the cas
 | Field             | Type            | Description
 |-------------------|-----------------|---------------------
 | relation          | ValueSet or null        | Type of relative location 
-| start             | int             | Start of subject within object. May be negative if upstream 
-| end               | int             | End of subject within object 
-| length            | int             | Length of overlap
+| start             | int or null             | Start of subject within object. May be "?" if intronic
+| end               | int or null             | End of subject within object.  
+| length            | int  or null          | Length of overlap. May be negative if boundary is not known
```

Comment:
small correction

```suggestion
| length            | int  or null          | Length of overlap. May be null if boundary is not known
```

Reply:
A similar correction regarding the "?" on the previous line 😉